custom-connectors/inQubaJourney/apiDefinition.swagger.json

```diff
@@ -0,0 +1,301 @@
+{

```

Comment:
@prinayinq Please let us know why do you want to open source your connector as a custom connector. Please note that we can not deploy and publish custom connectors in any test or production enviornment.

Reply:
Hi. This was a requirement for us as part of the ISV Connect Program. Additionally the intent is to provide our customers with seamless integration to inQuba within existing ecosystems they may have and drastically reduce project onboarding.

Reply:
@sriyen-msft the connector would need to be available when a user searches "InQuba Journey" from power automate. Is this not the route for that?

Reply:
I've uploaded the files under certified-connector. Please process as a certified connector and not custom

Reply:
I have also removed the code from the custom connector folder and only left under certified connector